ament_python/script_dir.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Override the setup.py command line argument --script_dir."""
+
+import os
+import sys
+
+
+def install_scripts_to_libexec(package_name):
+    """Override sys.argv to install scripts into lib/<pkgname> instead."""
+    try:
+        prefix_index = sys.argv.index('--prefix')
+    except ValueError:
+        return
+    prefix = sys.argv[prefix_index + 1]
```

Comment:
Had an offline discussion with @dirk-thomas about this, and decided it should either fallback to some default if `--prefix` is not passed or it should error out to avoid the prefix and the target for the scripts from having different base paths.